XPNet.CLR/Scenery/XPlaneScenery.cs

```diff
@@ -172,9 +213,21 @@ public unsafe void Dispose()
 			PluginBridge.ApiFunctions.XPLMDestroyProbe(m_probeRef);
 		}
 
-		public unsafe IXPProbeResult ProbeTerrainXYZ(float inX, float inY, float inZ)
+		public unsafe IXPProbeInfo ProbeTerrainXYZ(float inX, float inY, float inZ)
 		{
-			return new XPProbeResult(m_probeRef, inX, inY, inZ);
+			return new XPProbeInfo(m_probeRef, inX, inY, inZ);
 		}
 	}
+
+	/// <summary>
+	/// The result of the probing
+	/// </summary>
+	public enum XPProbeResult : int
+	{
+		// MAINT: This needs to be kept in sync with the
+		// XPLMProbeResult enum from the X-Plane API
+		ProbeHitTerrain = 0,
+		ProbeError = 1,
+		ProbeMissed = 2
```

Comment:
Same note here as elsewhere: I think these can just be HitTerrain, Error and Missed, b/c in .NET you already have to include the enum name to reference an enumeration, so you've already typed Probe once anytime you use these.  So like XPProbeResult.HitTerrain instead of XPProbeResult.ProbeHitTerrain.  (I could see this going either way, and reasonable people could disagree, this is more an argument for keeping the API consistent -- doing it the same way everywhere).

Reply:
In fact, I had your previous comment in mind here, and I was hesitating. The thing here is that the class is a ProbeResult, and not a probe. So if we had an enumerable "ProbeResultOK", "ProbeResultError" or so, I would totally agree, and I would name it "OK" or "Error". But here I understand it in the way of "The probe result is, that the probe hit terrain", as it is also explicated in the SDK documentation. For the drawing phases the intention was clearer, because they separated the word "Phase" from the actual phase, that is, for example, xplm_Phase_FirstScene. In a way, the naming of the enumerables is not 100% consistent in the SDK, because I would have felt a more consistent way of enumerating this to be "xplm_Result_ProbeHitTerrain", "xplm_Result_ProbeMissed", and "xplm_Result_ProbeError". In the end I am totally okay with whatever you feel is best. Would you still prefer if I remove "Probe" here?